docs/src/man/guide.md

```diff
@@ -25,3 +25,198 @@ object acts like an array of bytes; this involves some overhead so it may not be
 
 (Help in writing more detailed documentation would be welcome!)
 
+## Multiple Sockets
+Many ZMQ applications use multiple sockets.
+Unfortunately, integrating the ZMQ with the Julia task scheduler can be tricky and there currently is not one optimal way to use multiple sockets.
```

Comment:
This is not true. There is one optimal way (using one-Task-per-stream) and a suboptimal way (polling) which is simpler but scales poorly (because any blocking calls on one stream will cause all other streams to be unresponsive)